app/config-perfparser.h.in

```diff
@@ -0,0 +1,2 @@
+#cmakedefine01 HAVE_ZSTD
```

Comment:
missing license header? even for an autogenerated file we need that afaiu?

Reply:
OK, done.